runtime/kubernetes/codegen/doc.go

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) 2022 Target Brands, Inc. All rights reserved.
+//
+// Use of this source code is governed by the LICENSE file in this repository.
+
+// Package codegen is a dummy package to help go mod know that
+// we need k8s.io/code-generator.
+//
+// This package is not meant to be imported anywhere.
+package codegen
+
+import (
+	_ "k8s.io/code-generator"
```

Comment:
golangci is complaining:
> blank-imports: a blank import should be only in a main or test package, or have a comment justifying it (revive)

Is there a cleaner way to make sure we have "k8s.io/code-generator"? We don't need it at runtime, just when generating the deepcopy methods and the clientset for the custom CRD.